provider/schema/execution.go

```diff
@@ -267,24 +268,28 @@ func (e *ExecutionData) resolveResources(ctx context.Context, meta ClientMeta, p
 	shouldCascade := parent == nil
 	var err error
 	resources, err = e.copyDataIntoDB(ctx, resources, shouldCascade)
+	totalCount := uint64(len(resources))
```

Comment:
`copyDataIntoDB` always returns `nil` resources if returning with an `err`.

Reply:
so why not move totalCount lengh thing after the iferr? if err != nil, totalCount will always be zero, so why not do return 0, err below?

Reply:
correct fixed